### PR TITLE
add a11y auditing

### PR DESCRIPTION
**Pull request by stacylondon**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -17,8 +17,8 @@ site/
 # coverage reports
 coverage/
 
-# test reports
-test-reports/
+# a11y reports
+a11y-audit.json
```
Comment (alexreardon) [inline .gitignore]: can we put this with our other reports?

File: .circleci/config.yml [hunk 2]
```diff
@@ -133,6 +133,25 @@ jobs:
...
+            - v7-dependencies-{{ checksum "yarn.lock" }}
+
+      # PR's from forks cannot use the dependency cache for performance reasons
+      - run:
+          name: 'Forked PR dependency install'
+          command: yarn
+
+      - run:
+          name: Accessibility Audit
+          command: node browser-test-harness.js yarn test:accessibility
```
Comment (alexreardon) [inline .circleci/config.yml]: can we get the json output added as an artifact?

Comment (stacylondon) [reply .circleci/config.yml]: done!

File: .gitignore (hunk 1 above)
Comment (stacylondon) [reply .gitignore]: done!

File: package.json [hunk 3]
```diff
@@ -33,6 +33,8 @@
     "prettier_target": "*.{js,jsx,md,json} src/**/*.{js,jsx,md,json} test/**/*.{js,jsx,md,json} docs/**/*.{js,jsx,md,json} stories/**/*.{js,jsx,md,json} cypress/**/*.{js,jsx,md,json}"
   },
   "scripts": {
+    "test:accessibility": "lighthouse http://localhost:9002/iframe.html?id=single-vertical-list--basic --chrome-flags='--headless' --output=json --output-path=./test-reports/lighthouse/a11y-audit.json && yarn test:accessibility:report",
+    "test:accessibility:report": "node a11y-audit-parse.js",
```
Comment (alexreardon) [inline package.json]: does this need to be a separate step?

Comment (stacylondon) [reply package.json]: At first I was thinking I might have `test:accessibility` and `test:accessibility:ci` and then they could share that common step but now there is only a need for `test:accessibility` so I'll consolidate and just have the one NPM script for this.

Comment (stacylondon) [reply package.json]: Done; just a single step now.

**Pull request merged**